packages/adp-tooling/src/types.ts

```diff
@@ -31,19 +32,24 @@ export interface AdpWriterConfig {
         reference: string;
         layer?: UI5FlexLayer;
         title?: string;
+        content?: Content[];
```

Comment:
You are only using it if it is a cloud system (if I read the code correctly) which I think is a bit odd because the API does not indicate that.

Reply:
Content is used by both onPrem and Cloud if it's provided